### ISSUE TITLE
Add custom tag to be included in the text

### ISSUE THREAD

**Issue by ltGuillaume** [andreskrey/readability.php#81]
TorrentFreak.com articles are sometimes polluted with `<g></g>` tags by the spell checker they use. Readability consistently filters out all words encapsulated by these tags, so I'd like to prevent this. How should I do this properly?

I've already tried adding `|g` to the `okMaybeItsACandidate` regexp in `NodeUtility.php`, and tried to add `'g',` to `$phrasing_elems` in `NodeTrait.php`. Neither of these operations did the trick.

(I am using Readability as part of [Tiny Tiny RSS](https://git.tt-rss.org/fox/tt-rss/src/master/vendor/andreskrey/Readability))

**Comment by andreskrey** [comment]
I'm not able to reproduce this with version 2.0.1

I tried using [this article](https://torrentfreak.com/three-pirate-tv-device-sellers-jailed-for-a-total-of-17-years-190321/) and all the words wrapped within `<g>` tags were in the resulting HTML.

I'm not sure what version they use in Tiny Tiny RSS because instead of using a dependency manager they just pull and commit the whole source code of readability (which is a _terrible_ approach to manage dependencies). Mind trying again but using version v2.0.1 locally? I suspect they are using an old version, a modified one or something else.

**Comment by ltGuillaume** [comment]
Sh*t, you're right, so sorry. In the current configuration tt-rss didn't even use Readability to extract the full article. Going through the tt-rss code again (it ain't "Tiny Tiny", I can tell you that!) I found that Readability actually is only used when the plugin for it is enabled, otherwise it's got its own article extraction methods. I made a stupid assumption based on what I read in the code of a recent commit, turns out it isn't like I expected. Sorry to have put the blame on Readability ;) ;)
